### PR TITLE
[JB][SUPP-432]

### PR DESCRIPTION
**Pull request by barnesjake**
- add some tests
- configure what files/packages should be ignored by scoverage
- delete some unused stuff

### CODE REVIEW

File: test/controllers/DetermineTaxIdControllerSpec.scala [hunk 1]
```diff
@@ -170,7 +170,43 @@ class DetermineTaxIdControllerSpec extends ItSpec {
       }
 
     }
+  }
+
+  "AuthenticatedActionRefiner bonus tests" - {
```
Comment (darrell-plant) [inline test/controllers/DetermineTaxIdControllerSpec.scala:175]: bonus tests?

File: project/ScoverageSettings.scala [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import sbt.Def
+import scoverage.ScoverageKeys
+
+object ScoverageSettings {
+  lazy val scoverageSettings: Seq[Def.SettingsDefinition] = {
+    val excludedFiles = Seq(
+      """<empty>""",
+      """Reverse.*""",
```
Comment (AndyHWChung) [inline project/ScoverageSettings.scala:8]: would this not be covered already if we exclude the `*.Reverse.*` package below?

Comment (barnesjake) [reply project/ScoverageSettings.scala:8]: Good point, I'll check

File: test/controllers/DetermineTaxIdControllerSpec.scala (hunk 1 above)
Comment (barnesjake) [reply test/controllers/DetermineTaxIdControllerSpec.scala:175]: Haha sorry, poorly named but I thought I'd sneak into this class as that's the controller being used to test - I didn't want to create a whole new spec just for a few tests to cover off AuthenticatedActionRefiner

Comment (darrell-plant) [reply test/controllers/DetermineTaxIdControllerSpec.scala:175]: Maybe better to have an `AuthenticatedActionRefinerSpec` though (some time)

Comment (barnesjake) [reply test/controllers/DetermineTaxIdControllerSpec.scala:175]: Sure, I'll add now

Comment (darrell-plant) [reply test/controllers/DetermineTaxIdControllerSpec.scala:175]: I'm not sure if this is a good example, but ... https://github.com/hmrc/vmv-frontend/blob/a53755bd9ec661597d615ed59bd587f2fb97f714/test/uk/gov/hmrc/vmv/actions/CcaAccountActionRefinerSpec.scala

Comment (barnesjake) [reply test/controllers/DetermineTaxIdControllerSpec.scala:175]: It does look like it, but Pawel is _very_ against mockito 😂

**Comment by AndyHWChung** [comment]
doing a spec for this would be a quick win:
image
it's a relatively simple controller endpoint where you just test the content

**Comment by barnesjake** [comment]
Sure, will add now

File: test/controllers/DetermineTaxIdControllerSpec.scala (hunk 1 above)
Comment (darrell-plant) [reply test/controllers/DetermineTaxIdControllerSpec.scala:175]: is he against unit tests 🙂 ?

**Review by AndyHWChung — APPROVED** [review]

**Pull request merged**